### PR TITLE
PHPTRN-7,OOP classes,methods

### PR DESCRIPTION
**Pull request by MihailBoicko**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -224,3 +224,76 @@
...
+//    public $password = "Пароль";
+//    public $email = "Емейл";
+//    public $city = "Город";
+//}
+//
+//$admin = new User();
+//$user1 = new User();
+//
+//$admin->name = "Alexey";//Названия свойств пишутся без знака $(->name)
+//$user1->name = "Andrey";
```
Comment (revakirill) [inline index.php:300]: spaces not needed according to autoformat?

File: index.php [hunk 2]
```diff
@@ -224,3 +224,76 @@
...
+//            echo "Hello {$this->name}";
+//    }*/
+//
+//    function getInfo() {//если метод без спецификатора ,автобатически public;
+//        return "{$this->name}. {$this->sername}";
+//    }
+//
+//}
+//$admin = new User();
+//$admin->name = "Alexey";
```
Comment (revakirill) [inline index.php:327]: can we set values via setters and retrieve them via getters?

**Review by revakirill — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (MihailBoicko) [reply index.php:300]: According to the auto-format there are no spaces, but I think I'd better put them in to make the code look readable

File: index.php (hunk 2 above)
Comment (MihailBoicko) [reply index.php:327]: I guess it's a leading question, but I haven't heard anything about it. I'll find out

**Review by revakirill — APPROVED** [review]

**Pull request merged**